docs/api/connectivity/bluetooth/Optimising_for_performance.md

```diff
@@ -0,0 +1,173 @@
+# Optimising application for throughput and power consumption
+
+BLE devices are usually battery powered so performance might mean different things in different applications. You have
+to decide what is more important in yours - minimising power consumption on one or both sides of the communication or
+maximising throughput and/or latency. Some optimisation steps can in fact achieve both.
+
+This guide will discuss some trade-offs that should be considered and best practices that improve performance on all
+fronts.
+
+## Power consumtpion
+
+Any radio activity will consume power. Depending on what the stack is doing you have to power the radio even when no
+data is being sent. It is important to understand when radio is active.
+
+### Connections
+
+The most intuitive power consumption rate to understand is when using connections. Each take turns sending and receiving
+packets at set interval.
+
+
+    CENTRAL
+    ┌────┐ ┌────┐           ┌────┐ ┌────┐         ┌────┐ ┌────┐ ┌────┐ ┌────┐
+    │send│ │recv│           │send│ │recv│         │send│ │recv│ │send│ │recv│
+    └────┘ └────┘           └────┘ └────┘         └────┘ └────┘ └────┘ └────┘
+      connection interval
+     ◄─────────────────────►
+
+    PERIPHERAL
+    ┌────┐ ┌────┐                                 ┌────┐ ┌────┐ ┌────┐ ┌────┐
+    │recv│ │send│                                 │recv│ │send│ │recv│ │send│
+    └────┘ └────┘                                 └────┘ └────┘ └────┘ └────┘
+      slave latency
```

Comment:
Arm has a "progressive terminology" commitment to remove terms like "slave" from its products and documentation. If there is a straightforward alternative here then it may be worth considering. However we need to ensure any alternative is readily understood in context, and the value slaveLatency seems to be used in the API, so any change should be consistent.

Reply:
Slave latency is part of BLE spec and is the term used in all other tutorials and guides.

Reply:
As Paul mentioned it is the official terminology from the Bluetooth organization. It is a shame as they don't describe the connection and master/slave but instead use the terms central/peripheral and left slave latency in there.